### DISCUSSION TITLE
Event tracking without page load tracking

### DISCUSSION THREAD

**Discussion by mlissner** [plausible/analytics#1917] [Anything Else]
We don't use plausible on all our pages because some of them get a lot of traffic, which gets very expensive to track, but we'd still like to use the event tracking when people click links on those pages.

It [looks like](https://plausible.io/docs/custom-event-goals) to enable event tracking you need to put the usual plausible JS on your page, which would automatically track it (and run up charges for us).

Is it possible to do event tracking on a page without logging that the page was loaded?

**Comment by mlissner** [comment]
Hm, I just found this page, which sort of looks like it could work: https://plausible.io/docs/excluding-pages

I guess we could exclude a viral page by doing:

```
<script 
  defer 
  data-domain="yourdomain.com" 
  src="https://plausible.io/js/script.hash.exclusions.outbound-links.js"
  data-exclude="/**></script>
<script>window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }</script>
```

And then 

```
onclick=plausible("clicked it")
```

Does that make sense?

**Reply by metmarkosaric** [reply]
We also have the manual extension which could help: https://plausible.io/docs/script-extensions#scriptmanualjs

**Reply by mlissner** [reply]
Neat, thanks. That's cool.
